docs/thinkst_canary.md

```diff
@@ -0,0 +1,77 @@
+# Thinkst Canary API
+
+## Canarytoken Factory
+
+To facilitate the automation of Canary token creation without requiring the
+use of the main auth token a factory auth token can be used, which is a limit
+use key that can only be used to create other tokens.
+
+An existing factory auth token can be used with;
+
+```ruby
+factory = ThinkstCanary::Factory.new(
+  factory_auth: <factory_auth_token>,
+  flock_id: 'flock:123...'
+)
+```
+
+**TODO:** For a `factory` as defined above, query the API to fetch the `flock_id` and `memo`.
```

Comment:
Should this be done: example snippet for this would be good, or link to relevant documentation

Reply:
There isn't currently a way to fetch details of a factory auth token from the API. I raised this via their support email and they say they are looking into it.

I put this TODO here because I think it would be useful to do but the API doesn't support it yet.